content/guides/core/reports/collaborate-on-reports.md

```diff
@@ -8,14 +8,30 @@ title: Collaborate on reports
 weight: 30
 ---
 
-Once you have saved a report, you can select the **Share** button to collaborate. A draft copy of the report is created when you select the **Edit** button. Draft reports auto-save. Select **Save to report** to publish your changes to the shared report.
+This page describes various ways to collaborate on reports with your team.
 
-A warning notification will appear if an edit conflict occurs. This can occur if you and another collaborator edit the same report at the same time. The warning notification will guide you to resolve potential edit conflicts.
+## Share a report
+When viewing a report, click **Share**, then:
+1. To share a link to the report with an email address or a username, click **Invite**. Enter an email address or username, select **Can view** or **Can edit**, then click **Invite**. If you share by email, the email address does not need to be a member of your organization or team.
```

Comment:
Replace ordered list with unorderd list? These points are not steps, but rather user stories based on how they want to share a report.

Reply:
Done